README.md

```diff
@@ -36,27 +37,33 @@ By default the helpers builds bare for every architecture for both iOS and Andro
 
 ```sh
 # iOS simulator only
-hello-pear --ios-sim
+hello-pear -c --ios-sim
 
 # iOS simulator only with x64 architecture
-hello-pear --ios-sim x64
+hello-pear -c --ios-sim x64
 
 # iOS and iOS simulator
-hello-pear --ios --ios-sim arm64
+hello-pear -c --ios --ios-sim arm64
 
 # Android only arm archs
-hello-pear --android arm64 arm
+hello-pear -c --android arm64 arm
 
 # Android only arm64
-hello-pear --android arm64
+hello-pear -c --android arm64
 
 # Android only x86 archs
-hello-pear --android x64 ia32
+hello-pear -c --android x64 ia32
```

Comment:
@saimonmoore This section is about building, not necessarily configuring. Note the quoted section above that states the first time you run you need to call with `--configure`. I think that section could be clearer about supporting configuring for only a select set of targets, but it does cover the fact that you need to configure the first time and does default to configuring all targets.